### PR TITLE
Minor formatting changes for README.md

### PR DESCRIPTION
**Pull request by dancrossnyc**
Just to make this slightly easier to read in the terminal:
add line breaks and replace HTML-style <tt> tags to back
ticks, which expand to <code> sequences.

Signed-off-by: Dan Cross <cross@gajendra.net>

### CODE REVIEW

File: sprog.c [hunk 1]
```diff
@@ -582,6 +570,12 @@ main(int argc, char *argv[])
 	return EXIT_SUCCESS;
 }
 
+int
+isvowel(int c)
```
Comment (arnoldrobbins) [inline sprog.c:574]: Please change the return type on this to be `bool`.

**Review by arnoldrobbins — COMMENTED** [review]
Looks very nice. Please:
1. Make the change requested in the review.
2. Update the ChangeLog also.
And then I will merge the PR.

Thanks for the effort!

File: sprog.c (hunk 1 above)
Comment (dancrossnyc) [reply sprog.c:574]: Done.

**Comment by dancrossnyc** [comment]
Updated ChangeLog and made the requested change. Thanks!

**Pull request merged**